app/javascript/dashboard/modules/conversations/components/MessageContextMenu.vue

```diff
@@ -62,6 +62,9 @@ export default {
       getAccount: 'accounts/getAccount',
       currentAccountId: 'getCurrentAccountId',
     }),
+    conversationPanelElement() {
+      return document.querySelector('.conversation-panel');
```

Comment:
Is there a ref available instead? 
Instead of this, I guess at the top level we can provide the `conversationPanelElement` and then inject it downstream whenever needed. `https://vuejs.org/guide/components/provide-inject`

Reply:
@scmmishra Done https://github.com/chatwoot/chatwoot/pull/11454/commits/46f193765d955f049b0dae1dbf703dfedcf57e33